addon/models/node.js

```diff
@@ -167,11 +167,11 @@ export default OsfModel.extend(FileItemMixin, {
         return promise;
     },
 
-    addChild(title, description, category, isPublic) {
+    addChild(title, description=null, category='project', isPublic) {
```

Comment:
I went ahead and removed the default value for isPublic so as to not change the existing behavior and potentially break things. It is always specified when called in Quick Files move to project anyways.

Reply:
Works for me 👍